tfhe/benches/integer/bench.rs

```diff
@@ -952,11 +1178,14 @@ criterion_group!(
 
 criterion_group!(
     default_parallelized_ops,
+    neg_parallelized,
+    abs_parallelized,
     add_parallelized,
     sub_parallelized,
     mul_parallelized,
     div_parallelized,
     rem_parallelized,
+    div_rem_parallelized,
```

Comment:
thanks for adding the benchmarks, here I believe we may want only the div_rem version